### DISCUSSION TITLE
Is it possible with SqlBoiler to insert a `Team` with it's `has-many` `Players` using a single insert?

### DISCUSSION THREAD

**Discussion by frederikhors** [aarondl/sqlboiler#960] [Q&A] [answered]
Is it possible with SqlBoiler to insert a `Team` with it's `has-many` `Players` using a single insert?

Example:

```go
team := Team{
  name: "Real"
  players: []Player{
    {name: "John"},
    {name: "Frank"}
  }
}

err := team.Insert(ctx, db, boil.Infer())
```

Is this possible?

**Comment by aarondl** [comment, marked as answer]
There's no bulk-insertion supported by sqlboiler right now, they will always all be a separate query.

You could try `team.AddPlayers(..., true)` which will insert them for you.
